### DISCUSSION TITLE
Array searching in JSONB not working in sequelize

### DISCUSSION THREAD

**Discussion by HMhamedminaee** [sequelize/sequelize#16297] [Help & Questions]
Hey Sequelize team 
I have a JSONB object in Postgres 
```
{
    "searches": [
        {
            "details": {
                "statistics": [
                    {
                        "name": "logs_searched",
                        "unit": "undefined",
                        "value": 3870
                    },
                    {
                        "name": "byte_size",
                        "unit": "byte",
                        "value": 7467689
                    },
                    {
                        "name": "bytes_read",
                        "unit": "byte",
                        "value": 6519584
                    },
                    {
                        "name": "duration",
                        "unit": "ms",
                        "value": 177
                    }
                ]
}

```
now I am planning to search for example any rows that has a duration greater than 160. If it was not wrapped in an array I could do this easily 

```
    const querystrOp: any = {
      where: {
           "details.statistics.duration":  { [Op.gt]:160 }
       }
     }
```
    
but since statistic value is in a form of array It fails. Any way I can get this to working? or this is not supported by sequelize?

**Comment by ephys** [comment]
You can only do [simple property or index navigation](https://sequelize.org/docs/v7/querying/json/), like you'd do in JavaScript. If you need more than this, you'll need to use one of the [JSON function or operators](https://www.postgresql.org/docs/current/functions-json.html) that postgres supports using [raw SQL](https://sequelize.org/docs/v7/querying/raw-queries/)
